app/src/main/java/com/babylon/wallet/android/presentation/settings/personas/personadetail/PersonaDetailScreen.kt

```diff
@@ -258,15 +265,22 @@ private fun PersonaDetailList(
                         },
                         dApp = dApp
                     )
-                    Spacer(modifier = Modifier.height(dimensions.paddingLarge))
+                    Spacer(modifier = Modifier.height(RadixTheme.dimensions.paddingDefault))
+                }
+            }
+            item {
+                GrayBackgroundWrapper(
+                    modifier = Modifier.fillMaxWidth()
+                ) {
+                    Spacer(modifier = Modifier.height(RadixTheme.dimensions.paddingXXXLarge))
                 }
             }
         }
         item {
-            Spacer(modifier = Modifier.height(dimensions.paddingDefault))
+            Spacer(modifier = Modifier.height(RadixTheme.dimensions.paddingDefault))
             WarningButton(
                 modifier = Modifier
-                    .padding(horizontal = dimensions.paddingDefault),
+                    .padding(horizontal = RadixTheme.dimensions.paddingDefault),
```

Comment:
warning button should be higher according to zeplin

Reply:
It is the same as all the action buttons throughout the app. Maybe we should update all the buttons in a separate PR.

Reply:
It looks like all the buttons in zeplin are a bit higher I think (primary, secondary and warning - as we call them), so yes, it make sense to do so